wca-regulations.md

```diff
@@ -88,7 +88,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 2i2b) The competitor must not interact with (e.g. operate, hold, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).
 - 2j) The WCA Delegate may disqualify a competitor from a specific event.
     - 2j1) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event.
-        - 2j1a) The results of all remaining attempts in the event are recorded as DNF.
+        - 2j1a) If the competitor has already finished one or more attempts in a round after being disqualified, the results of all remaining attempts in the event are recorded as DNF. If the competitor has finished no attempts, no results should be recorded.
```

Comment:
```suggestion
        - 2j1a) If the competitor has already finished one or more attempts in a round after being disqualified, the results of all remaining attempts in the event are recorded as DNF. If did not start any attempt, no results should be recorded.
```

I think this would should @xsrvmy's concerns.

Reply:
The first part of the sentence should also started?

Reply:
I'm sorry but I don't understand what you mean. Do you mean about my proposal or the original proposal (Pedro's one)?

Reply:
I think what @xsrvmy means is that in `If the competitor has already finished one or more attempts in a round after being disqualified` (the first part of my initial proposal), "finished" should also be altered to "started". I'll push a commit that consolidates both of these changes.